03. Condicionais e Listas/desafio-05/app.js

```diff
@@ -15,7 +15,7 @@ new Vue({
 		}
 	},
 	methods: {
-		isArray (valor) {
+		checkIfisArray (valor) {
```

Comment:
```suggestion
		isArray(valor) {
```
Voltar o nome para **isArray** e tirar o espaço entre o nome da função e os parâmetros da mesma.

Reply:
Feito!